CMakeLists.txt

```
@@ -947,9 +947,9 @@ if (LLAMA_LTO)
 endif()
 
 if (LLAMA_CCACHE)
-    find_program(LLAMA_CCACHE_FOUND ccache)
-    if (LLAMA_CCACHE_FOUND)
-        set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+    find_program(LLAMA_CCACHE_PATH ccache)
+    if (LLAMA_CCACHE_PATH)
+        set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${LLAMA_CCACHE_PATH})
```

Should this be in quotes?
```suggestion
        set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${LLAMA_CCACHE_PATH}")
```